citestwheel.Dockerfile

```
@@ -103,6 +104,17 @@ python -m pip install "rapids-dependency-file-generator==1.*"
 pyenv rehash
 EOF
 
+RUN <<EOF
+# Install OpenTelemetry instrumentation
+pip install opentelemetry-distro[otlp] opentelemetry-exporter-prometheus
+curl -L -o "otel-cli-${CPU_ARCH}.tar.gz" https://github.com/equinix-labs/otel-cli/releases/download/v0.4.5/otel-cli_0.4.5_linux_${CPU_ARCH}.tar.gz
+tar -zxf  "otel-cli-${CPU_ARCH}.tar.gz"
+mv otel-cli /usr/local/bin/
+git clone -b add-conda-build-instrumentation https://github.com/msarahan/opentelemetry-python-contrib
```

```suggestion
git clone -b add-conda-build-instrumentation https://github.com/rapidsai/opentelemetry-python-contrib
```
we should probably fork this internally to RAPIDS somehow instead of using my personal fork.